traits/MLControl.php

```diff
@@ -180,6 +186,18 @@ protected function makeRenderFormField()
         return $field;
     }
 
+    public function getLocaleFieldName($code)
+    {
+        if ($this->isLongFormNeeded()) {
+            $names = HtmlHelper::nameToArray($this->formField->arrayName);
+            $name = $this->formField->getName('RLTranslate[' . $code . '][' . implode('][', $names) . ']');
+        } else {
+            $name = $this->formField->getName('RLTranslate['.$code.']');
```

Comment:
Can we rename the prefix to WinterTranslate yet or would that be a breaking change?

Reply:
I'd have to check, but I would make a special PR for this.